internal/cephfs/controllerserver.go

```diff
@@ -21,7 +21,9 @@ import (
 	"errors"
 	"fmt"
 
+	"github.com/ceph/ceph-csi/internal/cephfs/core"
```

Comment:
isn't this `core "github.com/ceph/ceph-csi/internal/cephfs/core"` ?

Reply:
Do we need an alias?

Reply:
its not needed as there are no other core packages we are importing in controllerserver.